packages/esm-patient-medications-app/src/medications/active-medications.component.tsx

```diff
@@ -27,9 +32,19 @@ const ActiveMedications: React.FC<ActiveMedicationsProps> = ({ patientUuid, show
     isValidating,
   } = usePatientOrders(patientUuid, 'ACTIVE', config.careSettingUuid);
 
+  const { currentVisit } = useVisitOrOfflineVisit(patientUuid);
+
   const launchOrderBasket = React.useCallback(() => {
```

Comment:
This whole construct is 4 times the same. Should it be in a separate function (`useLaunchOrderBasket`)?

Reply:
Done!